src/libmongoc/doc/includes/bulk-remove-one-opts.txt

```diff
@@ -5,5 +5,5 @@
 
 ``opts`` may be NULL or a BSON document with additional command options:
 
-* ``collation``: Configure textual comparisons. See :ref:`Setting Collation Order <setting_collation_order>`, and `the MongoDB Manual entry on Collation <https://www.mongodb.com/docs/manual/reference/collation/>`_. Collation requires MongoDB 3.2 or later, otherwise an error is returned.
+* ``collation``: Configure textual comparisons. See `Setting Collation Order <setting_collation_order>`_, and `the MongoDB Manual entry on Collation <https://www.mongodb.com/docs/manual/reference/collation/>`_. Collation requires MongoDB 3.2 or later, otherwise an error is returned.
```

Comment:
```suggestion
* ``collation``: Configure textual comparisons. See `Setting Collation Order <setting_collation_order_>`_, and `the MongoDB Manual entry on Collation <https://www.mongodb.com/docs/manual/reference/collation/>`_. Collation requires MongoDB 3.2 or later, otherwise an error is returned.
```

Based on our discussion in Slack it seems like the additional underscore might be necessary.